src/test_workflow/test_kwargs.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Filename should match class. Add a test file, even if you don't implement a test (you should).

Reply:
Fixed the name and added tests.